linux_os/guide/system/accounts/accounts-restrictions/root_logins/no_direct_root_logins/ansible/shared.yml

```diff
@@ -10,4 +10,6 @@
 
 - name: "Direct root Logins Not Allowed"
   shell: echo > /etc/securetty
+  args:
+    warn: False
```

Comment:
```suggestion
    executable: /bin/bash
    warn: False
```

Reply:
@shawndwells no `|` so don't need to add executable here.